tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -1,18 +1,23 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+# This block is used by testing and the SpECTRE command line interface.
 Executable: CharacteristicExtract
 Testing:
   Check: parse
   Priority: High
 
 ---
+# Start of the input file that controls the CCE evolution.
 
 Evolution:
+  # The initial sizes aren't super important because we use error-based adaptive
+  # time stepping to adjust the step size.
   InitialTimeStep: 0.25
   InitialSlabSize: 10.0
 
 ResourceInfo:
+  # Can ignore this section since CCE performs best on a single core.
   AvoidGlobalProc0: false
   Singletons:
```

Comment:
Change this to

```yaml
ResourceInfo:
  AvoidGlobalProc0: false
  Singletons: Auto
```

which is equivalent to what is below, but much more succinct and less confusing for users.